external/systemjs/plugin-babel-cached.js

```diff
@@ -12,6 +12,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* eslint-disable no-var */
+
```

Comment:
I'm purposely ignoring this file, since it's slated for removal (pending Firefox changes) in PR #12563.